cmd/antrea-agent/agent.go

```diff
@@ -160,7 +165,16 @@ func run(o *Options) error {
 		wireguardConfig,
 		networkReadyCh,
 		stopCh,
-		features.DefaultFeatureGate.Enabled(features.AntreaProxy))
+		features.DefaultFeatureGate.Enabled(features.AntreaProxy),
+		enableBridge)
+	if enableBridge {
+		// Restore network config before shutdown. ovsdbConnection must be alive when restore.
+		defer agentInitializer.RestoreOVSBridge()
```

Comment:
Could we cover the Pod crash case (could we recover host network in that case)?

Reply:
Do you mean agent Pod crash? It's OK for SIGTERM/SIGKILL on anrea-agent, thus the Pod crash should be OK.